test/error.js

```diff
@@ -33,4 +34,14 @@ describe('error handling', () => {
     assert.throws(humanizing({ language: null }), Error)
     assert.doesNotThrow(humanizing({ language: 'es' }), Error)
   })
+
+  it('does not throw an error when passed a bad language in the function and a valid fallback specified', () => {
+    function humanizingWith(options) {
```

Comment:
Again, could you fix the spacing here?

```suggestion
    function humanizingWith (options) {
```